ajax/libs/vue-lazyload/package.json

```diff
@@ -0,0 +1,27 @@
+{
+  "name": "vue-lazyload",
+  "filename": "vue-lazyload.js",
+  "version": "1.0.6",
+  "description": "Vue module for lazy-loading images in your vue.js applications.",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/hilongjw/vue-lazyload.git"
+  },
+  "keywords": [
+    "vue-lazyload",
+    "vue",
+    "lazyload",
+    "vue-directive"
+  ],
+  "author": "Awe <hilongjw@gmail.com>",
+  "license": "MIT",
+  "npmName": "vue-lazyload",
+  "npmFileMap": [
+    {
+      "basePath": "",
+      "files": [
+        "vue-lazyload"
```

Comment:
Use `vue-lazyload*.+(js|css|map)` or `vue-lazyload*` can get the file.
Thank you